components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -1812,125 +1816,341 @@ private int handleStartIndexEqualsNULL(Integer startIndex) {
         return startIndex;
     }
 
-    private List<Object> listGroups(Map<String, Boolean> requiredAttributes) throws CharonException {
+    /**
+     * Method to list all the groups.
```

Comment:
Better to start with `list..`

Reply:
Fixed: 0811c8c408e0d35596e71d82f167868b3f079ae7